src/components/attendee/NavOverlay/NavOverlay.tsx

```diff
@@ -54,24 +54,24 @@ export const NavOverlay: React.FC<navOverlayProps> = ({
   };
 
   return (
-    <div className={styles.navOverlay}>
-      <div className={styles.navOverlay__close} onClick={onClose}>
+    <div className={styles.NavOverlay}>
+      <div className={styles.NavOverlay__close} onClick={onClose}>
         Close
-        <span className={styles.navOverlay__close_icon} />
+        <span className={styles.NavOverlay__close_icon} />
```

Comment:
Another way of doing this would be something like https://github.com/sparkletown/sparkle/blob/078e19679552cdcf3101ae73445970c0430980d7/src/pages/StylePoC/util.ts as used in https://github.com/sparkletown/sparkle/pull/2719/files#diff-e9d8c07016b1c8410f2eb46b3d0783d6fb1ec10cb62db315cd3cdeb9f4887029R47-R54

Reply:
done in [020d1cd](https://github.com/sparkletown/sparkle/pull/2813/commits/020d1cd7ecd97a0ef06cbc5406d93673b6ecef72)

used `className('class_name', SN.className') in all places